content/tutorials/token-authentication.textile

```diff
@@ -207,11 +243,44 @@ blang[ruby].
 
   "See this step in Github":https://github.com/ably/tutorials/commit/5575d73
 
+blang[php].
+  By adding the following script, it is now ready to service clients wanting to authenticate with Ably.
+
+  Create @auth.php@ script:
+
+  ```[php]
+    require_once 'ably.php'
+    header('Content-Type: application/json');
+    # Issue token requests to clients sending a request to the /auth.php endpoint
+    echo json_encode($ably->auth->createTokenRequest()->toArray());
+  ```
+
+  Try running your server again with @php -S 0.0.0.0:8000@ (if it's not up already) and visiting "http://localhost:8000/auth.php":http://localhost:8000/auth.php and you should see a JSON token request in the form:
+
+  ```[json]
+  {
+    "keyName": "I2ACJQ.Lv8AUQ",
+    "ttl": 3600000,
+    "timestamp": 1473894038255,
+    "capability": "{\"*\":[\"*\"]}",
+    "nonce": "f6799a8e7fa6f77b8e1dac55314789b1",
+    "mac": "35nifY9SRZ8KRDfKOPIS1qYWGP16r2lD59zJo9TH8pA="
+  }
+  ```
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/f08a34f
+
 h2. Step 5 - Setting up a client to use token authentication
 
 Now that we have a web server running and an authentication endpoint, we are ready to setup the Ably Realtime client that uses token auth.
 
-First we need to set up a simple HTML page that will be served by the web server. Create the following HTML file named @index.html@ in the same folder as @server.js@:
+blang[nodejs].
+  First we need to set up a simple HTML page that will be served by the web server. Create the following HTML file named @index.html@ in the same folder as @server.js@:
+blang[ruby].
+  First we need to set up a simple HTML page that will be served by the web server. Create the following HTML file named @index.html@ in the same folder as @server.rb@:
+blang[php].
```

Comment:
Same comment as the other PR. Are you running this locally and checking that the page renders as expected.  This is necessary please.